### PR TITLE
New apps

### PR DESCRIPTION
**Pull request by OnGle**
adds prestashop and oscommerce

### CODE REVIEW

File: src/apps/prestashop/mod.rs
Comment (a3s7p) [inline src/apps/prestashop/mod.rs]: intended?

File: src/apps/prestashop/mod.rs
Comment (a3s7p) [inline src/apps/prestashop/mod.rs]: intended?

File: src/apps/oscommerce/mod.rs
Comment (a3s7p) [inline src/apps/oscommerce/mod.rs]: for oscommerce

File: src/apps/prestashop/mod.rs
Comment (a3s7p) [inline src/apps/prestashop/mod.rs]: for prestashop

**Review by a3s7p — APPROVED** [review]
a few nitpicks, otherwise lgtm

File: src/apps/prestashop/mod.rs
Comment (OnGle) [reply src/apps/prestashop/mod.rs]: This is what you get when you try out the new ``cargo clippy --fix``. In my head I was thinking "I know there's nothing dangerous it could do, warning for unnecessary var/clone is a great use case for this". I didn't think it would generate provably dead code, how silly.

File: src/apps/prestashop/mod.rs
Comment (OnGle) [reply src/apps/prestashop/mod.rs]: Good catch, removed.

File: src/apps/oscommerce/mod.rs
Comment (OnGle) [reply src/apps/oscommerce/mod.rs]: Can you tell I've been copying "core/mod.rs"?

**Comment by OnGle** [comment]
result type hasn't been updated either, I might fix that and add a few more appliances

**Comment by JedMeister** [comment]
> result type hasn't been updated either, I might fix that and add a few more appliances

Ok @OnGle - I'll leave this one for now and check in again on Monday (unless I hear otherwise).

**Comment by OnGle** [comment]
Return types fixed, added bookstack and mantis as well.

**Comment by JedMeister** [comment]
@OnGle Please feel free to let me know if/when this is ready to merge... :smile: (my comment re leaving until next week was on the assumption that you had likely finished for the week...).

**Comment by OnGle** [comment]
@JedMeister it should be good to go, I've addressed all the minor concerns noted above and I won't have anything else by this week, so unless you want to wait for another review now's as good as any :)

**Comment by JedMeister** [comment]
Awesome, thanks mate. Merging now...

**Pull request merged**
